content/en/security/cloud_siem/entities_and_risk_scoring.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Cloud SIEM Risk Based Entity Insights
+---
+
+{{< callout url=""https://datadoghq.com/private-beta/risk-based-entity-insights" header="Join the Beta!">}}
+Risk Based Entity Insights is in private beta.
+{{< /callout >}}
+
+## Overview
+
+[Cloud SIEM's Risk Based Entity Insights][4] consolidates multiple data sources, such as SIEM threats and CSM insights, into a profile representing a single security entity, such as an IAM user.
+
+With Risk Based Entity Insights, you can:
+
+- Explore entities, filtering them by attributes such as [risk score](#risk-scoring) or entity type.
+- View all data relevant to an entity, such as signals, misconfigurations, and identity risks.
+- Triage relevant items in bulk.
+- Take mitigation steps such as creating an automated workflow or creating a global case for an entity.
+
+## Explore Entities
+
+### Query and filter entities
+
+On the [Entities Explorer page][4], you can view all entities that have at least one signal.
+
+{{< img src="security/entities/entities-explorer.png" alt="A list of entities and their risk scores in the Entities Explorer" style="width:100%;" >}}
+
+### Quickly build context on an entity
+
+Click an entity in the [Explorer][4] to open the entity details panel.
+
+{{< img src="security/entities/entity-side-panel.png" alt="The details panel for an entity" style="width:90%;" >}}
```

Comment:
<img width="1500" alt="image" src="https://github.com/DataDog/documentation/assets/147530771/41412965-7e07-48b1-8c8a-bbef6e939eda">
use something like this from demo org instead :)

Reply:
hey jen sorry to bring this up again could you please use an entity side panel that has those high signals at the top (there is currently a bug that we are fixing but the impact on risk score should be sorted within the signals)